elements/pfe-progress-indicator/src/pfe-progress-indicator.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Progress Indicator",
+  "description": "This element creates a progress indicator when the user is waiting on http events.",
+  "type": "object",
+  "tag": "pfe-progress-indicator",
+  "class": "pfe-progress-indicator",
+  "category": "content",
+  "properties": {
+    "slots": {
+      "title": "Slots",
+      "description": "Definition of the supported slots",
+      "type": "object",
+      "properties": {
+        "content": {
+          "title": "Content",
+          "type": "array",
+          "namedSlot": true,
```

Comment:
This should be set to false and the oneOf should be "raw" instead of "progress-message".  The array inside content indicates what components or code is allowed inside the slot.  Since all HTML is allowed inside the slot, we can just use raw.